uqcsbot/poll.py

```diff
@@ -0,0 +1,82 @@
+import discord
+from discord import app_commands
+from discord.ext import commands
+
+from typing import Any
+
+# The unicode points for each number emoji starting from 0 to 10
+EMOJI_NUMBERS = [
+    "\u0030\u20E3",
+    "\u0031\u20E3",
+    "\u0032\u20E3",
+    "\u0033\u20E3",
+    "\u0034\u20E3",
+    "\u0035\u20E3",
+    "\u0036\u20E3",
+    "\u0037\u20E3",
+    "\u0038\u20E3",
+    "\u0039\u20E3",
+    "\U0001F51F",
+]
```

Comment:
Imo, rather than unicode these could just be the the Discord emoji names `one`, `two`, `three`, etc.